website/docs/r/vpc_ipv6_cidr_block_association.html.markdown

```diff
@@ -10,8 +10,8 @@ description: |-
 
 Provides a resource to associate additional IPv6 CIDR blocks with a VPC.
 
-When a VPC is created, a primary IPv6 CIDR block for the VPC must be specified.
-The `aws_vpc_ipv6_cidr_block_association` resource allows further IPv6 CIDR blocks to be added to the VPC.
+When a VPC is created, a primary IPv4 CIDR block for the VPC must be specified.
```

Comment:
I don't think this sentence is necessary. It doesn't add to the understanding of the resource being documented.